File: python/problem-16(XJumps)/XJUMP.py
# cook your dish here
t=int(input())
for i in range(t):
    (x,y) = map(int,input().split())
    if x%y==0:
        num_step=int(x/y)
        print(num_step)
    else:
        step=int(x/y)
        t=x%y
        total_step = step+t
        print(total_step)

File: c++/problem-15(XJumps)/XJUMP.cpp
#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int t;
	cin>>t;
	while (t--){
	    int x,y;
	    cin>>x>>y;
	    if (x%y==0){
	        int num_step=x/y;
	        cout<<num_step<<endl;
	    }
	    else{
	        int step=x%y;
	        int t=x/y;
	        int total_step=step+t;
	        cout<<total_step<<endl;
	    }
	}
	return 0;
}
